tools/android_sdk/create_cipd_packages.sh

```diff
@@ -15,6 +15,7 @@ print_usage () {
   echo ""
   echo "This script downloads the packages specified in packages.txt and uploads"
   echo "them to CIPD for linux, mac, and windows."
+  echo "To confirm you have write permissions run 'cipd acl-check flutter/android/sdk/all/ -writer'."
```

Comment:
Can this be done as part of this script?

Reply:
It can but honestly given we dont have a dry run. One of the only(and easy) ways to verify what is about to be uploaded is to comment out the `rm -rf $upload_dir` and the upload command and see what is about to be uploaded. Adding that check would add another piece of code to comment out. I added this documentation so that when I come back to do https://github.com/flutter/flutter/issues/138021 I know what is required to warn on missing permissions. But in a dry run we would continue anyway since the user is not trying to upload.